packages/bridge/src/components/Withdraw/Withdraw.tsx

```diff
@@ -138,54 +245,91 @@ export const Withdraw: React.FC<WithdrawProps> = () => {
+          <div style={{ display: 'flex', alignItems: 'center' }}>
+            <Typography variant={'h6'}>
+              <b>ADD NOTE</b>
+            </Typography>
+          </div>
+          <RelayerSettings
+            role='button'
+            aria-disabled={!activeChain}
+            onClick={() => {
+              console.log('open relayer settings modal');
             }}
-            inputProps={{ style: { fontSize: 14 } }}
-            fullWidth
-            placeholder={`Enter account address`}
-          />
-          <FormHelperText error={Boolean(validationErrors.recipient && recipient)}>
-            {validationErrors.recipient}
-          </FormHelperText>
-        </InputLabel>
-      </InputSection>
-
-      <SpaceBox height={16} />
-
+            className='select-button'
+          >
+            <SettingsIcon />
+            <p style={{ fontSize: '14px', color: '#B6B6B6', marginLeft: '5px' }}>RELAYER</p>
+          </RelayerSettings>
+        </div>
+        <div className='note-input'>
+          <BridgeNoteInput error={note ? validationErrors.note : ''} value={note} onChange={setNote} />
+        </div>
+      </WithdrawNoteSection>
       {depositNote && (
-        <>
-          <RelayerInput
-            tokenSymbol={depositNote?.note.tokenSymbol || ''}
-            feesGetter={feesGetter}
-            relayers={relayersState.relayers}
-            setActiveRelayer={setRelayer}
-            relayerApi={relayerApi}
-            activeRelayer={relayersState.activeRelayer}
-          />
+        <AddressAndInfoSection>
+          <div style={{ padding: '10px 35px' }}>
+            <Typography variant={'h6'}>
+              <b>RECIPIENT ADDRESS</b>
+            </Typography>
+          </div>
+          <div className='address-input'>
+            <InputBase
+              value={recipient}
+              onChange={(event) => {
+                setRecipient(event.target.value as string);
+              }}
+              inputProps={{ style: { fontSize: 14 } }}
+              fullWidth
+              placeholder={`Please paste your address here`}
+            />
+            <FormHelperText error={Boolean(validationErrors.recipient && recipient)}>
+              {validationErrors.recipient}
+            </FormHelperText>
+          </div>
+          <SpaceBox height={16} />
+          <div className='information-item'>
+            <p className='title'>Deposit Amount</p>
+            <p className='value'>
+              {depositNote.note.amount} {depositNote.note.tokenSymbol}
+            </p>
+          </div>
+          <div className='information-item'>
+            <p className='title'>Chains</p>
+            <p className='value'>
+              {getChainNameFromChainId(parseChainIdType(Number(depositNote.note.sourceChainId)))}
+              {` -> `}
+              {getChainNameFromChainId(parseChainIdType(Number(depositNote.note.targetChainId)))}
+            </p>
+          </div>
```

Comment:
I agree it would be better to have as much text in an <h*> tag as we can manage for standardizing across the app, but I think this should be done in a different PR.